src/CTA.WebForms2Blazor/ControlConverters/GridViewControlConverter.cs

```diff
@@ -21,7 +72,32 @@ public GridViewControlConverter() : base()
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            node.OwnerDocument.OptionOutputOriginalCase = true;
+            
```

Comment:
Similar to above, add comment explaining that we're explicitly calling this to preserve capitalization of attribute and element names